cob_monitoring/src/hz_monitor.py

```diff
@@ -142,9 +141,8 @@ def publish_diagnostics(self, rt_HZ_store = []):
                     publishing_rate_error = True
                     consolidated_error_messages["publishing rate is too high for topics"].append(topic)
                 else:
-                    if not publishing_rate_error:
-                        hz_status.level = DiagnosticStatus.OK
-                        hz_status.message = 'all publishing rates are ok'
+                    hz_status.level = DiagnosticStatus.OK
+                    hz_status.message = 'all publishing rates are ok'
```

Comment:
I guess the part
`if not publishing_rate_error:`
is not useless in the case where one hz_monitor instance monitors the publish rate of several topics...because with your changes, it would set `DiagnosticStatus.OK` although maybe some other topics are "too low" or "too high"...